test/testdata/infer/generics/disabled/variance_methods.rb

```diff
@@ -0,0 +1,45 @@
+# typed: true
+
+class A
+  extend T::Sig
+  extend T::Helpers
+  extend T::Generic
+
+  abstract!
+
+  Key = type_member(:in) # error: Classes can only have invariant type members
+  Val = type_member(:out)
```

Comment:
I expected this to be an error too

Reply:
Yes, me too. Oddly it only raises an error for the first one.